tools/toolchains/mbed_toolchain.py

```diff
@@ -872,7 +872,7 @@ def add_regions(self):
                     "s" if len(regions) > 1 else "",
                     ", ".join(r.name for r in regions)
                 ))
-                self._add_all_regions(regions, "MBED_RAM")
+                self._add_all_regions(regions, "MBED_RAM_APP")
```

Comment:
I'm not keen to make this change yet. While we certainly will get to a point where we have application's ram separated from physical ram, we have no configuration for that use right now. Your proposal to use `mbed_ram_start` and `mbed_ram_size` for this purpose will:
 1) Break any project that depends on those configuration settings current behavior.
 2) Be inconsistent with `mbed_rom_start` and `mbed_rom_size`

Reply:
This is to fix duplicate `MBED_RAM_START` symbol in `.profile-ld`. Would you point out another solution to it?